ENGAGEHF/Home.swift

```diff
@@ -55,17 +59,28 @@ struct HomeView: View {
                 .tabItem {
                     Label("Education", systemImage: "brain")
                 }
+            NavigationStack {
+                DevicesTab(appName: ENGAGEHF.appName ?? "ENGAGE") {
+                    Text("Hold down the Bluetooth button for 3 seconds to put the device into pairing mode.")
+                }
+                    .environment(\.advertisementStaleInterval, 15)
```

Comment:
Nice way to configure this. It might even make sense to make these modifiers a dedicated named modifier. So far I have seen SwiftUI mainly doing their own modifiers for key elements that modify the view tree.

Reply:
We can migrate these options to be a single SwiftUI modifier (e.g., `View/bluetoothScanningOptions(minimumRSSI: advertisementStaleInterval:)`). Might make it slightly more concise to specify both options.